reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -305,7 +305,7 @@ class StaticHtmlReporter : Reporter {
                 }
             }
             td {
-                p { +ruleViolation.violation.message }
+                markdown(ruleViolation.violation.message)
```

Comment:
@bennati Per https://github.com/oss-review-toolkit/ort/blob/main/model/src/main/kotlin/RuleViolation.kt#L53 message is a plain text string and not a MarkDown string as RuleViolation.HowToFix. 

If I remember it correctly we made message plain text so 1) can be easily ingested by tools and 2) printing to logs but @mnonnenmacher should weigh as he has worked the most on the Evaluator.

Reply:
I am a bit undecided, it would be nice to be able to format the messages, but it also has benefits to keep them simple strings. In any way, if this is changed, it should also be changed for the web app reporter for consistency. I think this would be a good topic to discuss in the dev meeting to get more opinions.

Reply:
I think you misunderstand my request: I am not interested in enabling markdown in the violation message, e.g. displaying links and such. I want to fix an issue I am having with normal text not being displayed correctly.

For example the following HTML source text
```
line 1

line 2
```

is currently displayed as follows: `line 1 line 2`.

I would expect newlines not to be ignored.

Reply:
> In any way, if this is changed, it should also be changed for the web app reporter for consistency.

Agreed. And shouldn't we support Markdown then also for issues?

Reply:
I've updated the commit as I have found that marking the message as 'unsafe' allows to add new lines without the need to enable markdown.
We can still discuss whether to enable markdown or not, but that should be independent from this PR, I think.

Reply:
> > In any way, if this is changed, it should also be changed for the web app reporter for consistency.
> 
> Agreed. And shouldn't we support Markdown then also for issues?

@bennati As you are on it, could you do the same change for issue messages as part of this PR?